### PR TITLE
Prepare for Azure.Batch release 5.1.0

### PR DESCRIPTION
**Pull request by matthchr**
- This includes support for the new reactivate task API, as well as a freshly generated protocol level with the Nightly20160824 AutoRest build.
- Also includes a few documentation fixes

### CODE REVIEW

File: src/Batch/Client/Src/GeneratedProtocol/AccountOperations.cs [hunk 1]
```diff
@@ -1,58 +1,33 @@
-// Copyright (c) Microsoft and contributors.  All rights reserved.
```
Comment (cormacpayne) [inline src/Batch/Client/Src/GeneratedProtocol/AccountOperations.cs]: @matthchr was the license header removed from these files intentionally? If not, @tbombach has anything changed to cause this?

Comment (matthchr) [inline src/Batch/Client/Src/GeneratedProtocol/AccountOperations.cs]: Ah... I think I know what happened.  That's a really good catch @cormacpayne I will fix it right now.

Comment (matthchr) [inline src/Batch/Client/Src/GeneratedProtocol/AccountOperations.cs]: Should be fixed now

**Comment by cormacpayne** [comment]
LGTM

**Pull request merged**